local_planner/src/nodes/planner_functions.cpp

```diff
@@ -172,7 +173,7 @@ void generateNewHistogram(Histogram& polar_histogram,
   // Normalize and get mean in distance bins
   for (int e = 0; e < GRID_LENGTH_E; e++) {
     for (int z = 0; z < GRID_LENGTH_Z; z++) {
-      if (counter(e, z) > 0) {
+      if (counter(e, z) > n_points_occupied) {
```

Comment:
This should be `>=`, so that n_points_occupied = 1 works with a single point in the cell.